internal/runner/operation_release.go

```diff
@@ -148,9 +151,27 @@ func (r *Runner) executeReleaseOp(
 			// Mark for deletion
 			ds = append(ds, d)
 		}
+		rl, err := r.client.ListReleases(ctx, &pb.ListReleasesRequest{
+			Application: app.Ref(),
+			Workspace:   project.WorkspaceRef(),
+		})
+		// Set up a slice of releases, whose max length is the # of releases
+		// we have
+		rs := make([]*pb.Release, 0, len(rl.Releases))
+		var rc int
+		for _, release := range rl.Releases {
+			for _, d := range ds {
+				if release.DeploymentId == d.Id {
+					rs = append(rs, release)
+					rc++
```

Comment:
We could always use `len(rs)` later rather than capturing its length with this intermediary var!